.gitignore

```diff
@@ -6,7 +6,6 @@ target/
 build/
 
 # IDEA
-*.iml
 .idea/
```

Comment:
Why did you make this change?

Reply:
*.iml is covered by .idea/, so it's redundant, anyhow, I reversed it